torch/csrc/jit/frontend/sugared_value.h

```diff
@@ -507,6 +507,36 @@ struct TORCH_API CastValue : public BuiltinFunction {
   TypePtr type_;
 };
 
+struct TORCH_API TensorCastValue : public SugaredValue {
```

Comment:
It's good to kill the old `_cast_` operators but  not great it needs a new SugaredValue type. Is it possible to implement these in builtin_functions.cpp and then return the actual function as a method? That would reduce the number of concepts.